pkg/expapi/types.go

```diff
@@ -134,3 +136,40 @@ type HorizontalPodAutoscalerList struct {
 
 	Items []HorizontalPodAutoscaler `json:"items"`
 }
+
+type DerivedNodeMetrics struct {
+	api.TypeMeta   `json:",inline"`
+	api.ObjectMeta `json:"metadata,omitempty"`
+
+	NodeMetrics      MetricsWindows            `json:"nodemetrics"`
+	SystemContainers []DerivedContainerMetrics `json:"systemcontainers"`
+}
+
+type DerivedContainerMetrics struct {
+	Name    string         `json:"name"`
+	Windows MetricsWindows `json:"windows"`
+}
+
+type DerivedNodeMetricsList struct {
+	api.TypeMeta `json:",inline"`
+	api.ListMeta `json:"metadata,omitempty"`
+
+	Items []DerivedNodeMetrics `json:"items"`
+}
+
+// Last overlapping 10s, 1m, 1h and 1d as a start
+// Updated every 10s, so the 10s window is sequential and the rest are
+// rolling.
```

Comment:
Explain what are the underlying measurements over which mean, max, and 95 are computed.
Are they 10secondly integrations of CPU usage, I assume?

I assume the 10s window will only have Mean, since there is just one 10s measurement?
And the 1m will not have 95%ile since you can't do a good 95%ile of only 6 measurements?

Do you want to make Mean, Max and NinetyFive be *ResourceUsage instead of ResourceUsage, so that it is clear which are not set?